packages/core/test/integration/query-interface.test.js

```diff
@@ -570,8 +576,8 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
       this.User = this.sequelize.define('users', {
```

Comment:
I'll review the tests below here after you've resolved the conflicts with the main branch